src/apps/ENGINE/compiler.h

```diff
@@ -176,6 +177,9 @@ class COMPILER : public VIRTUAL_COMPILER
     ATTRIBUTES *rAP;
 
   public:
+    std::shared_ptr<spdlog::sinks::basic_file_sink_mt> error_warning_sink;
+    std::shared_ptr<spdlog::logger> tracelog, errorlog, warninglog;
```

Comment:
I'd suggest putting each variable declaration on a separate line.

Reply:
Be done.